Description: Include libphp*-embed ld library path when building PHP plugin.
Author: Janos Guljas <janos@debian.org>
Last-Update: 2016-02-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/plugins/php/uwsgiplugin.py
+++ b/plugins/php/uwsgiplugin.py
@@ -16,6 +16,7 @@
 
 CFLAGS = [os.popen(PHPPATH + ' --includes').read().rstrip(), '-Wno-sign-compare']
 LDFLAGS = os.popen(PHPPATH + ' --ldflags').read().rstrip().split()
+LDFLAGS.append('-L/usr/lib/php' + php_version)
 
 if ld_run_path:
     LDFLAGS.append('-L%s' % ld_run_path)
